webapp/src/ts/modules/contacts/contacts-content.component.html

```diff
@@ -99,7 +99,7 @@ <h3>{{group?.type?.group_key | translate}}</h3>
         <ul class="action deceased">
           <li *ngIf="group?.deceasedCount > 0" class="content-row deceased">
             <a [routerLink]="['/contacts', selectedContact?.doc?._id, 'deceased']">
-              <span translate [translateParams]="{ count: group.deceasedCount }">contact.deceased.view</span>
+              <span translate [translateParams]="{ count: group.deceasedCount | localizeNumber }">contact.deceased.view</span>
```

Comment:
Just to double check, do we want to use the `localizeNumber` for `field.value` of contact summary? 
It can contain numbers, right?

Reply:
But maybe we don't want to convert phones or external id... I'm going to leave the question here just in case other numeric field 🤔

Reply:
In contact summary, the filter is available directly via the config: https://github.com/medic/cht-core/blob/master/config/default/contact-summary.templated.js#L22
There's actually more control over contact summary fields.